tx/textual/valuerenderer/valuerenderer.go

```diff
@@ -51,6 +51,19 @@ func (r Textual) GetValueRenderer(fd protoreflect.FieldDescriptor) (ValueRendere
 			return intValueRenderer{}, nil
 		}
 
+	case fd.Kind() == protoreflect.StringKind:
+		return stringValueRenderer{}, nil
+
+	case fd.Kind() == protoreflect.MessageKind:
```

Comment:
Merge with the name-based dispatch introduced in #12860

In particular, this replace the current TODO comment for a default.